sdk/test/unit/introspection.test.ts

```diff
@@ -1,6 +1,13 @@
-import {getInterfaceId} from '../../src';
+import {
+  InvalidAddressError,
+  getInterfaceId,
+  getProtocolVersion,
+} from '../../src';
+import {TEST_HTTP_URI} from '../constants';
+import {mockContractProtocolVersion, mockJSONRPCProvider} from '../mocks';
 import {Interface} from '@ethersproject/abi';
 
+const ADDRESS_ONE = `0x${'0'.repeat(39)}1`;
```

Comment:
could we have this value in the constants file, to avoid having it twice. It was already created in the `ens.test.ts`.

Reply:
We said we don't want to have these constants being exported everywhere and actually are trying to get rid of them.

Reply:
Yes, it was in the constant files, but we agreed to stop creating this kind of constants, if you ckeck michaels comments you'll see

Reply:
is the test constants file exported?